fuzzaldrin-plus/index.d.ts

```diff
@@ -5,25 +5,32 @@
 
 export as namespace fuzzaldrin;
 
-export interface IQueryOptions {
+export interface QueryOptions {
```

Comment:
I understand that the `I` prefix might not be needed, but I'd recommend not changing this, simply for the sake of backward compatibility. No reason to make users using these typings have to change their code.